openpower/configs/zz_defconfig

```diff
@@ -29,6 +29,7 @@ BR2_PACKAGE_IPMITOOL_USB=y
 BR2_PACKAGE_MDADM=y
 BR2_PACKAGE_NCURSES_WCHAR=y
 BR2_PACKAGE_DROPBEAR=y
+BR2_PACKAGE_HCODE=n
```

Comment:
Same comment as openpower/configs/openpower_p9_mambo_defconfig

Reply:
true, no need for BR2_HCODE_INCLUDE_IONV to be on,  I can fix that up

Reply:
I haven't changed that in what I merged, but we can fix it up in subsequent patch.